plugin-app-back-end: Phase 1 deprecation (#26855)
backstage/backstage: .changeset/gentle-vans-learn.md

---
'@backstage/plugin-app-backend': patch
---

> Deprecated useRouter and alpha exports and removed deprecated dependency for @backstage/backend-commons

Reviewer: ```suggestion
Deprecated `createRouter`, alpha exports, and removed deprecated dependency on `@backstage/backend-common`
```
